### PR TITLE
WIP: Update enhancement for aws-load-balancer-operator

### PR DESCRIPTION
**Pull request by thejasn**
Update enhancement for `aws-load-balancer-operator`
---

- Updates some implementation details
- Updates test plan and current caveats in testing
- Removes non-applicable details for initial release

### CODE REVIEW

**Review by arjunrn — COMMENTED** [review]
Can you also update the implementation history section. In the drawbacks we should mention that services will also be reconciled. Also in the risk mitigation section for target type IP we can say that this won't work in OpenShift and so is not a risk.

File: enhancements/ingress/aws-load-balancer-operator.md [hunk 1]
```diff
@@ -317,44 +318,27 @@ does not remove the existing addon attachment from the provisioned load
...
+for the lb-controller won't be provisioned automatically, and will require manual
+intervention to do so. The required manual steps will be documented within the
+operator docs. Additional documentation for STS mode/manual mode can be found [here](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html)
+
+The lb-controller requires a validating and mutating webhook for correct operation.
+The operator will have to create the webhooks along with the controller deployment. 
+The webhook can be registered with a CA bundle which is used to verify the identity 
+of webhook by the API server. The [service-ca controller](https://docs.openshift.com/container-platform/4.10/security/certificate_types_descriptions/service-ca-certificates.html)
+can be used to generate certificates and have them injected into the webhook
+configurations. The operator will also watch the secret with the certificates so
```
[the remark this answers is not in the document]
Comment (thejasn) [reply enhancements/ingress/aws-load-balancer-operator.md]: Addressed.

**Comment by thejasn** [comment]
Closing in favor of https://github.com/openshift/enhancements/pull/1107

**Pull request closed without merging**